src/pages/dashboard/panels/ShokoNews.tsx

```diff
@@ -9,9 +9,21 @@ import { DashboardNewsType } from '@/core/types/api/dashboard';
 
 import { useGetShokoNewsFeedQuery } from '@/core/rtkQuery/externalApi';
 
+function isWithinXDaysFromToday(date: string) {
```

Comment:
Also either change the function name to `isWithin14DaysFromToday` or add the number of days as an argument with 14 as the default parameter.
It doesn't matter much because we are using it in a single place, but just to be correct.

Reply:
Originally, I was going to make it so you could specify how many days but then decided to hardcode it.